src/org/spdx/licenseTemplate/HtmlTemplateOutputHandler.java

```diff
@@ -98,17 +98,17 @@ private void removeEndParagraphTag() {
 	 */
 	public static String formatReplaceabledHTML(String text, String id) {
 		StringBuilder sb = new StringBuilder();
-		sb.append("\n<span ");
+		sb.append("\n<div ");
 		if (id != null && !id.trim().isEmpty()) {
 			sb.append("id=\"");
 			sb.append(escapeIdString(id));
 			sb.append("\" ");
 		}
 		sb.append("class=\"");
 		sb.append(REPLACEABLE_LICENSE_TEXT_CLASS);
-		sb.append("\">");
+		sb.append("\"  style=\"display: inline\">");
```

Comment:
With `<var>` for phrasing content, I don't think you need this style anymore.

Reply:
Would this be needed for the div's?

Reply:
> Would this be needed for the div's?

I don't think so, because all the inline cases will now be `var`.  This styling was a workaround for a case (inline `div`s) that should no longer happen.